sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -1996,6 +1996,27 @@ def message_content():
                     await receiver.complete_message(message)
                     count += 1
 
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedServiceBusResourceGroupPreparer(name_prefix="servicebustest")
+    @CachedServiceBusNamespacePreparer(name_prefix="servicebustest")
+    @ServiceBusQueuePreparer(name_prefix="servicebustest", dead_lettering_on_message_expiration=True)
+    @pytest.mark.parametrize("uamqp_transport", uamqp_transport_params, ids=uamqp_transport_ids)
+    @ArgPasserAsync()
+    async def test_async_queue_message_batch_large(
+        self, uamqp_transport, *, servicebus_namespace=None, servicebus_queue=None, **kwargs
+    ):
+
+        fully_qualified_namespace = f"{servicebus_namespace.name}{SERVICEBUS_ENDPOINT_SUFFIX}"
+        credential = get_credential(is_async=True)
+        async with ServiceBusClient(
+            fully_qualified_namespace, credential, logging_enable=False, uamqp_transport=uamqp_transport
+        ) as sb_client:
+            async with sb_client.get_queue_sender(servicebus_queue.name) as sender:
+                with pytest.raises(ValueError):
+                    batch_message = [ServiceBusMessage("test message " * 256 * 1024)]
```

Comment:
larger than 256 kb

Reply:
This test will fail in order to pass ?

we should add in tests for add as well.

Reply:
we have tests for batch for passing already

Reply:
there are ones that check for `MessageSizeExceededError` ? if so then cool :)

Reply:
https://github.com/Azure/azure-sdk-for-python/blob/480ed8878e88aba478ed6132def8d51bdf0cc909/sdk/servicebus/azure-servicebus/tests/test_message.py#L216 they catch ValueError which MessageSizeExceededError extends from